graphiql.go

```diff
@@ -4,21 +4,28 @@ import (
 	"encoding/json"
 	"html/template"
 	"net/http"
+	"strings"
 
 	"github.com/graphql-go/graphql"
 )
 
 // page is the page data structure of the rendered GraphiQL page
 type graphiqlPage struct {
-	GraphiqlVersion string
-	QueryString     string
-	ResultString    string
-	VariablesString string
-	OperationName   string
+	GraphiqlVersion              string
+	SubscriptionTransportVersion string
+	QueryString                  string
+	ResultString                 string
+	VariablesString              string
+	OperationName                string
+	EndpointURL                  template.URL
+	EndpointURLWS                template.URL
+	SubscriptionsEndpoint        template.URL
```

Comment:
Only `EndpointURLWS` is being used in the template, `SubscriptionsEndpoint` isn't. I personally favor the name `SubscriptionsEndpoint` and would pass `Endpoint` and `SubscriptionsEndpoint` (falling back to `Endpoint` if not set) in to the template.